build.gradle

```diff
@@ -760,6 +768,7 @@ task dockerUpload {
 }
 
 task manifestDocker {
+  dependsOn distDocker
```

Comment:
What changed to require this?

Reply:
Not sure about this, @jflo will hopefully know

Reply:
there was an implied dependency on distDocker already, Justin likely is just making it explicit.  manifestDocker needs the arch-specific docker images to be built before it can correctly create a multi-arch image